Formula/polymake.rb

```diff
@@ -0,0 +1,56 @@
+class Polymake < Formula
+  desc "Tool for computations in algorithmic discrete geometry"
+  homepage "https://polymake.org"
+  url "https://polymake.org/lib/exe/fetch.php/download/polymake-3.2r3.tar.bz2"
+  sha256 "8423dac8938dcd96e15b1195432f6a9844e8c34727c829395755a5067ce43440"
+
+  depends_on "ant"
+  depends_on "boost"
+  depends_on "gmp"
+  depends_on "mpfr"
+  depends_on "ninja"
```

Comment:
Are we sure this & `ant` are runtime deps? They almost never are.

Reply:
ninja is also necessary during runtime: missing C++-template instances are compiled when needed (second and third test in the formula check that this works)

I removed ant. Only needed for jreality, which is disabled anyway (as it does not work properly on macos)